src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput rightBeamBreak;
```

Comment:
These might be changed to the Playing With Fusion distance sensors with thresholds to be used as beam breaks, but that needs more testing before the testing is confirmed. Also the current plan is to actually have a front and back distance sensor, therefore when the front beam break sees a ring, then it pulls the ring back until the ring is seen by the back distance sensor. This would most likely need to be handled in `IntakeDefault`